napari/utils/events/event.py

```diff
@@ -673,20 +700,24 @@ class EmitterGroup(EventEmitter):
     """
 
     def __init__(
-        self, source=None, auto_connect=True, deprecated=None, **emitters
+        self,
+        source: Any = None,
+        auto_connect: bool = True,
+        deprecated: dict[str, str] | None = None,
+        **emitters: Type[Event] | EventEmitter | None,
```

Comment:
Not sure if you want to be consistant on wether  to use `|None`/`Optional`, here you use it, but higer you use `callback: Callback = None`, without the `|None`/`Optional` type annotations.

Reply:
yeah, now that I learned in the other PR that it's recommended... I'll default to that in the future